resources/guis.yml

```diff
@@ -44,7 +44,7 @@ guis:
   - Windows
   - Mac
   - Linux
-  price: Free / $29 / $49
+  price: Free for local and public repos / $59+/user annually (free 7-day trial)
```

Comment:
i understand your point here. the price was correct, but not many details. But is it relevant to add all these details? or can a user understand there is a free and paid version and then click to find more details? just to avoid cluttering the page with tons of text

see updated change:

<img width="697" alt="imagem" src="https://user-images.githubusercontent.com/1999050/178956005-b7a89927-eb86-4f99-b364-3f0eafc4de18.png">

vs current one

<img width="739" alt="imagem" src="https://user-images.githubusercontent.com/1999050/178956062-5b71a065-c043-4c62-b316-31b8ed2cc889.png">


so i'd say it's a matter of trade-off between having detailed info vs just the bare min and let users click to find more. For instance, would it be more interesting to have some expandable section which would add these extra details? or a on-hover thing? just wandering..

Reply:
You're absolutely right, I will change this to be shorter and less detailed. I like your idea of extra details hidden by default, but I think I'll keep that out of this PR which I prefer to just be a data update. Maybe I'll try adding that as a next PR :)